usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -831,14 +847,37 @@ def test_load_table_to_delta_for_sam_recipient(spark, s3_unittest_data_bucket, p
 )
 @mark.django_db(transaction=True)
 def test_load_table_to_delta_for_summary_state_view(
-    spark, s3_unittest_data_bucket, populate_usas_data, hive_unittest_metastore_db
+    spark, s3_unittest_data_bucket, populate_usas_data_and_recipients_from_broker, hive_unittest_metastore_db
 ):
+
+    # We need the award_search table to create the summary_state_view in delta
+    # And in order to create the award_search table, we need the following
+    load_delta_table_from_postgres("published_fabs", s3_unittest_data_bucket)
+    load_delta_table_from_postgres("detached_award_procurement", s3_unittest_data_bucket)
+
     tables_to_load = [
+        "awards",
+        "financial_accounts_by_awards",
+        "recipient_lookup",
+        "recipient_profile",
+        "sam_recipient",
+        "transaction_current_cd_lookup",
         "transaction_fabs",
         "transaction_fpds",
         "transaction_normalized",
+        "zips",
     ]
     create_and_load_all_delta_tables(spark, s3_unittest_data_bucket, tables_to_load)
+    verify_delta_table_loaded_to_delta(
+        spark, "award_search", s3_unittest_data_bucket, load_command="load_query_to_delta"
+    )
+
+    # We now want to load the award_search table that we created above along with other tables needed to create award_search
+    # Then create the summay_state_view table and populate it using the load_query_to_delta command
+    tables_to_load = ["transaction_fabs", "transaction_fpds", "transaction_normalized", "award_search"]
+    create_and_load_all_delta_tables(spark, s3_unittest_data_bucket, tables_to_load)
```

Comment:
It seems like there is some duplication going on here as far as what tables are created and loaded in spark. Can this be trimmed down at all?

Reply:
Also, we may be able to use the `load_dict_to_delta_table()` function in `spark_helpers.py` to mock out `award_search` in Spark instead of creating all those dependent tables. Each of those tables will require 2 extra spark jobs to be run (one for creation, one for population), which are pretty time consuming. 

I don't want this to hold up the PR because the PR is urgent, but this is something to think about it if it can be done quickly. You'd only need to mock out the fields required for the Summary State View query.

Reply:
I will create a tech debt ticket for this and link it here.